src/object/obj_tx.c

```diff
@@ -1793,8 +1801,16 @@ dc_tx_cpd_pack_sub_reqs(struct dc_tx *tx, tse_task_t *task, size_t size)
 	}
 
 	used = crp_proc_get_size_used(tx->tx_crt_proc);
-	D_ASSERTF(used <= size, "Input buffer size %ld is too small for real case %ld\n",
-		  size, used);
+	if (unlikely(used > size)) {
+		D_DEBUG(DB_TRACE, "Former estimated size %ld is too small, enlarge it to %ld\n",
+			size, used);
+
+		crt_proc_destroy(tx->tx_crt_proc);
```

Comment:
you need to check the return code for crt_proc_destroy.
coverity tool will complain at least.

Reply:
You are right. I will refresh the patch. Thanks